src/pages/login/forms/FormLogin.tsx

```diff
@@ -1,28 +1,30 @@
+import { Form } from "./Form";
 import { detect } from "detect-browser";
-import { useHistory } from "react-router-dom";
```

Comment:
Fairly certain removing this will cause it to fail to compile, I'll fix locally.